97.md

```diff
@@ -0,0 +1,25 @@
+NIP-97
+======
+
+Nostr Login
+-----------
+
+`draft` `optional`
+
+This NIP defines a method for a way for users to prove to apps that they control a certain private key. This can be used to enable login for services like the management panel of image hosts.
+
```

Comment:
Would it be a good idea to add a bit more detail on the intended use cases?

The first time I read the document it was not clear that this login is meant for non-Nostr apps/clients. I was confused for a minute thinking that this was attempting to be a way to login into a Nostr client (where multiple Nostr events need signing)

Reply:
I like "Login with Nostr" too